### DISCUSSION TITLE
These steps no longer work

### DISCUSSION THREAD

**Discussion by wj109s5** [jasontaylordev/CleanArchitecture#561] [General]
This documentation is old and the steps no longer work.  The front end will work, but any call to the middle tier results in a 404.

![image](https://user-images.githubusercontent.com/3514581/150704443-9d04ad8a-47a2-4869-a3b2-47628db14422.png)

Also what's this Duende noise about?  

![image](https://user-images.githubusercontent.com/3514581/150704506-0caa7e80-8605-4535-841d-d45312b5f4ff.png)

How many other things (undocumented) should I be finding licenses for and installing?  The YouTube didn't have any of this?

**Comment by wj109s5** [comment]
Also the blog post is very old and those steps no longer work either.

**Reply by y1nk** [reply]
I'm finding the same thing, quite frustrating
